Tools/MiniBrowser/wpe/main.cpp

```diff
@@ -276,8 +315,11 @@ static WebKitWebView* createWebView(WebKitWebView* webView, WebKitNavigationActi
 
 #if ENABLE_WPE_PLATFORM
     if (auto* wpeView = webkit_web_view_get_wpe_view(newWebView)) {
+        auto* wpeToplevel = wpe_view_get_toplevel(wpeView);
+        if (windowWidth > 0 && windowHeight > 0)
+            wpe_toplevel_resize(wpeToplevel, windowWidth, windowHeight);
```

Comment:
I don't think we should apply the size in this case, we are expected to use `WebKitWindowProperties` in this case, connecting to ready-to-show signal on the new web view. I see it's what we currently do, so it's probably ok to do this for the new API too, and handle the window properties in a follow up patch.

Reply:
As discussed, makes no sense to use the specified --size parameters for secondary windows that are opened.